subsys/net/lib/nrf_cloud/Kconfig.nrf_cloud_location

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2022 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+config NRF_CLOUD_CELL_POS
+	bool "Enable cellular positioning over MQTT via nRF Cloud Location"
+	select NRF_CLOUD_LOCATION
+
+config NRF_CLOUD_LOCATION
+	bool "Enable nRF Cloud Location (cellular and/or WiFi) over MQTT"
```

Comment:
OK, not sure about that commit. 

Anyways, was thinking if it would be possible that there is only NRF_CLOUD_LOCATION where NRF_CLOUD_REST and NRF_CLOUD_MQTT be used for flagging transport specifics.